ui/webui/src/components/installation/ReviewConfiguration.jsx

```diff
@@ -14,16 +14,125 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
-import React from "react";
+import cockpit from "cockpit";
+import React, { useEffect, useState } from "react";
 
 import {
-    Form
+    Checkbox, Title,
+    DataList, DataListItem,
+    DataListItemRow, DataListItemCells,
+    DataListCell,
+    DescriptionList, DescriptionListGroup,
+    DescriptionListTerm, DescriptionListDescription,
+    Flex, Popover,
 } from "@patternfly/react-core";
 
+import {
+    getSelectedDisks, getDeviceData,
+} from "../../apis/storage.js";
+
+import {
+    getLanguage, getLanguageData,
+} from "../../apis/localization.js";
+
+import { HelpIcon } from "@patternfly/react-icons";
+
+const _ = cockpit.gettext;
+
 export const ReviewConfiguration = () => {
+    const [deviceData, setDeviceData] = useState({});
+    const [selectedDisks, setSelectedDisks] = useState();
+    const [systemLanguage, setSystemLanguage] = useState("YYY");
+
+    useEffect(() => {
+        getLanguage()
+                .then(res => {
+                    getLanguageData({ lang: res }).then(res => {
+                        setSystemLanguage(res["native-name"].v);
+                    }, console.error);
+                }, console.error);
+        getSelectedDisks()
+                .then(res => {
+                    setSelectedDisks(res);
+                    // get detailed data for the selected disks
+                    res.forEach(disk => {
+                        getDeviceData({ disk })
+                                .then(res => {
+                                    setDeviceData(d => ({ ...d, [disk]: res[0] }));
+                                }, console.error);
+                    });
+                }, console.error);
+    }, []);
+
+    // handle case of disks not (yet) loaded
+    if (!selectedDisks) {
```

Comment:
Maybe do 
``` 
if (!selectedDisks || !lang) {
    return null;
}
```

Reply:
And remote the YYY default.

Reply:
I'm not sure its possible not to have a language set - but lets do that for consistency reasons. :)